apps/admin/frontend/src/screens/unconfigured_screen.tsx

```diff
@@ -149,6 +151,7 @@ export function UnconfiguredScreen(): JSX.Element {
       // TODO: read file content from backend
       const fileContent = await readFileAsyncAsString(file);
       await saveElectionToBackend(fileContent);
+      saveSystemSettingsToBackend(JSON.stringify(DEFAULT_SYSTEM_SETTINGS));
```

Comment:
Not directly related to this PR but as I was passing through all the machines' backend stores, I noticed that we delete old settings before loading new ones in VxCentralScan and VxMark, but not VxAdmin and VxScan.

_VxCentralScan and VxMark_

https://github.com/votingworks/vxsuite/blob/aa21fe78e1b1470613a434f6c5aca1d1d704173c/apps/central-scan/backend/src/store.ts#L184-L190

https://github.com/votingworks/vxsuite/blob/aa21fe78e1b1470613a434f6c5aca1d1d704173c/apps/mark/backend/src/store.ts#L107-L113

_VxAdmin and VxScan_

https://github.com/votingworks/vxsuite/blob/aa21fe78e1b1470613a434f6c5aca1d1d704173c/apps/admin/backend/src/store.ts#L230-L235

https://github.com/votingworks/vxsuite/blob/aa21fe78e1b1470613a434f6c5aca1d1d704173c/apps/scan/backend/src/store.ts#L1241-L1246

Maybe this doesn't matter much since, in order to upload new settings, you have to unconfigure a machine first, which presumably deletes the old settings. But we might want to be consistent.

Reply:
I agree we should be consistent: added in [b0e56a9](https://github.com/votingworks/vxsuite/pull/3345/commits/b0e56a9b4ab0c3a8bb38cf40f858a78f6ee8d763)